src/App.js

```diff
@@ -3,7 +3,7 @@ import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App"><Calculator screen="0" /></div>
+    <div className="App"><Calculator /></div>
   );
 }
 
```

Comment:
### `README.md` file
- [ ] To make your documentation looks more professional and descriptive, I think it would be better if  you update some of your sections by adding appropriate and correct information in each one, such as:
  * I can suggest you update the `Getting Started` section with good info of how to `clone` your repo, `access` the cloned directory, `install` dependencies, `run/use` your app, `run tests`(if exist),... To help you, just refer **[this gist_file](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**
  * In your ` Live Demo` section, you can add a working link(just refer this **[link](https://www.youtube.com/watch?v=OG71ARNRPT4) to help you how to deploy**) or you can remove it from now and add it later when you will deploy it.

Reply:
Thank you 😄  I now updated the README file with the suggested changes, do I need to request another review?